common/ethnicities/wc_ethnicities_troll.txt

```diff
@@ -0,0 +1,692 @@
+	}
+}
+
+sand_troll_ethnicity = {
+	template = "darkspear_trollish"
+	
+	skin_color = {
+		1 = { 0.224 0.129 0.298 0.239 } # Pale Yellow
+	}
+	eye_color = {
+        1 = { 0.686 0.310 0.686 0.350 } # Blue
+		1 = { 0.176 0.482 0.176 0.741 } # Yellow
+		1 = { 0.110 0.396 0.110 0.714 } # Orange/Brown
+	}
+	hair_color = {
+		1 = { 0.129 0.722 0.129 0.867 } # Brown
+		1 = { 0.153 0.537 0.153 0.682 } # Yellow
+		1 = { 0 0.643 0.043 0.800 } # Orange/Red
+	}
+}
+
+ice_troll_ethnicity = {
+	template = "darkspear_trollish"
+	
+	skin_color = {
+		1 = { 0.741 0.494 0.741 0.529 } # Blue
+		1 = { 0.659 0.396 0.659 0.427 } # Icy Blue
+		1 = { 0.714 0.459 0.714 0.459 } # Pale Blue
+	}
+	eye_color = {
+        1 = { 0 0.502 0 0.502 } # Blue
+	}
+	hair_color = {
+		5 = { 0.737 0.094 0.737 0.094 } # White
+		5 = { 0.749 0.510 0.749 0.686 } # Dark Blue
+		5 = { 0.741 0.310 0.741 0.765 } # Gray Blue to Dark Blue
+		1 = { 0.914 0.596 0.914 0.639 } # Purple
+		1 = { 0.584 0.510 0.584 0.620 } # Sea-Green
+		1 = { 0.867 0.322 0.867 0.376 } # Light Pink
+		1 = { 0.902 0.420 0.902 0.498 } # Pink
+	}
+	
+	#Height
+	gene_height = {
+        1 = { name = normal_height  range = { 0.77 0.8 }      }
+    }
+	
+	###Jaw
+	gene_jaw_width = {
+		1 = { name = jaw_width_pos    range = { 0.5 0.65 }    }
+	}
+	gene_bs_jaw_def = {
+		1 = { name = jaw_def_pos    range = { 0.6 1 }    }
+	}
+	
+	#Eyes
+	special_eyes = {
+        10 = { name = magic_eyes      range = { 0 1 } }
+    }
+	
+	#Chin	
+	gene_chin_width = {
+        1 = { name = chin_width_neg    range = { 0.5 1 }    }
+	}
+	
+	#Body
+	gene_bs_body_type = {
+		10 = { name = body_fat_head_fat_low		range = { 0.4 0.5 } }
+	}
+	gene_bs_body_shape = {
+		10 = { name = body_shape_triangle_half		range = { 0.6 1 } }
+	}
+	
+	###Other	
+	hairstyles = {
+		50 = { name = troll_hairstyles 		range = { 0.0 1.0 } }
+		5 = { name = northern_hairstyles 		range = { 0.376 0.376 } }
+		1 = { name = no_hairstyles 		range = { 1 1 } }
+	}
+}
+
+dark_troll_ethnicity = {
+	template = "darkspear_trollish"
+	
+	skin_color = {
+		1 = { 0.737 0.314 0.769 0.353 } # Pale Blue
+		1 = { 0.831 0.373 0.831 0.396 } # Dark Purple
+	}
+	hair_color = {
+		1 = { 0.733 0.863 0.733 0.992 } # Dark Blue to Black
+	}
+	
+	#Body
+	gene_bs_body_type = {
+		10 = { name = body_fat_head_fat_low		range = { 0.4 0.5 } }
+	}
+	
+	###Other	
+	hairstyles = {
+		1 = { name = troll_hairstyles 		range = { 0.0 1.0 } }
+	}
+}
+
+forest_troll_ethnicity = {
+	template = "ice_troll_ethnicity"
+	
+	skin_color = {
+		1 = { 0.349 0.729 0.349 0.776 } # Dark Green
+		5 = { 0.341 0.329 0.412 0.396 } # Pale Green
+	}
+	hair_color = {
+		15 = { 0.004 0.627 0.004 0.784 } # Red
+		10 = { 0.369 0.682 0.388 0.820 } # Green
+		5 = { 0.094 0.624 0.165 0.667 } # Orange/Yellow
+		5 = { 0.502 0.808 0.502 0.867 } # Cyan
+		1 = { 0 0.043 0 0.043 } # White
+	}
+	
+	#Height
+	gene_height = {
+        1 = { name = normal_height  range = { 0.73 0.78 }      }
+    }
```

Comment:
You can remove duplicate genes now since the ice trolls are base. Like they have the same jaws. Almost the same height. The same body type.

Reply:
> 
> 
> You can remove duplicate genes now since the ice trolls are base. Like they have the same jaws. Almost the same height. The same body type.

Removed all duplicates in 8bdb0de6d307090a2ef2ba00f1afc05f100fc3fc.